featureflags/analytics.py

```diff
@@ -160,7 +160,9 @@ def _send_data(self) -> None:
                 target_attributes: List[KeyValue] = []
                 if not isinstance(unique_target.attributes, Unset):
                     for key, value in unique_target.attributes.items():
-                        target_attributes.append(KeyValue(key, value))
+                        # Attribute values need to be sent as string to
```

Comment:
Note: I considered type checking the `value` but Google suggests its an expensive operation. So it is cheaper and just as valid to cast all values - even strings - to strings.